README.md

```diff
@@ -83,3 +87,4 @@ Support this project by becoming a sponsor. Your logo will show up here with a l
 [10]: https://github.com/libremesh/network-profiles/
 [12]: https://opencollective.com/libremesh
 [13]: https://libremesh.org/development.html
+[14]: https://github.com/irina556/lime-packages/blob/easing-libremesh-virtualization/VIRTUALIZING.md
```

Comment:
Hi! The link should point to the file that will be in the libremesh/lime-packages default branch repository.

Reply:
ping @irina556